src/lib/viewers/text/TextBaseViewer.js

```diff
@@ -47,36 +53,31 @@ class TextBaseViewer extends BaseViewer {
     /**
      * Zooms by increasing or decreasing font size
      * @public
-     * @param {string} inOrOut - in or out
+     * @param {string} type - in, out, reset
```

Comment:
Is reset currently a used type?

Reply:
I don't believe so, but it's supported implicitly in the existing code. This change just brings the function docs more in line with the other viewers.